src/BusinessLayer/Contracts/IUserService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+using BusinessLayer.Models;
+using JetBrains.Annotations;
+
+namespace BusinessLayer.Contracts
+{
+    public interface IUserService
+    {
+        [ItemNotNull]
```

Comment:
According to your logic [ItemCanBeNull] attribute should be used because you return null in case of wrong email or password